src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -1601,8 +1601,18 @@ private MemberDeclarationSyntax ParseTypeDeclaration(SyntaxListBuilder<Attribute
 
                     return this.ParseClassOrStructOrInterfaceDeclaration(attributes, modifiers);
 
-                case SyntaxKind.RefKeyword:
                 case SyntaxKind.StructKeyword:
+                    // report use of ref struct class
```

Comment:
should say readonly struct

---
In reply to: [118581738](https://github.com/dotnet/roslyn/pull/19713#discussion_r118581738) [](ancestors = 118581738)